commit 8810c1fc050347e1bef0d3eb89edd30c8e840648
Author: Ilya Tumaykin <itumaykin@gmail.com>
Date:   Sun Sep 17 21:54:37 2017 +0300

build: remove duplicate android option after 72a8120da

The first one (line 140) comes from 69650851f8 and is the correct one.
The second one (line 731) comes from 72a8120daa and slipped in with the
revert commit.

Remove the second one.

diff --git a/wscript b/wscript
index 9d885884d..9fe5b0bc8 100644
--- a/wscript
+++ b/wscript
@@ -727,10 +727,6 @@ video_output_features = [
         'desc': 'Direct3D support',
         'deps': [ 'win32-desktop' ],
         'func': check_cc(header_name='d3d9.h'),
-    }, {
-        'name': '--android',
-        'desc': 'Android support',
-        'func': check_statement('android/api-level.h', '(void)__ANDROID__'),  # arbitrary android-specific header
     }, {
         # We need MMAL/bcm_host/dispmanx APIs. Also, most RPI distros require
         # every project to hardcode the paths to the include directories. Also,
